tests/MavenTests/testCSVReports.cpp

```diff
@@ -396,57 +399,59 @@ void TestCSVReports::verifyUntargetedPeakReport(vector<mzSample*>& samplesToLoad
     mzUtils::splitNew(headersString, "," , header);
-    QVERIFY(header.size() == 21);
+    QVERIFY(header.size() == 22);
 
     // check if parent group values are correctly written
     vector<std::string> peakValues1;
     mzUtils::splitNew(peakString1, "," , peakValues1);
-    QVERIFY(peakValues1.size() == 21);
+    QVERIFY(peakValues1.size() == 22);
     QVERIFY(peakValues1[0] == "15");
     QVERIFY(peakValues1[1] == "210.150269@16.714417");
     QVERIFY(peakValues1[2] == "210.150269@16.714417");
     QVERIFY(peakValues1[3] == "");
-    // QVERIFY(peakValues1[4] == "testsample_2");
-    // QVERIFY(peakValues1[5] == "210.150375");
-    // QVERIFY(peakValues1[6] == "210.150452");
+    QVERIFY(peakValues1[4] == "");
+    // QVERIFY(peakValues1[5] == "testsample_2");
+    // QVERIFY(peakValues1[6] == "210.150375");
     // QVERIFY(peakValues1[7] == "210.150452");
-    // QVERIFY(peakValues1[8] == "16.710");
-    // QVERIFY(peakValues1[9] == "16.603");
-    // QVERIFY(peakValues1[10] == "17.392");
-    // QVERIFY(peakValues1[11] == "0.801");
-    // QVERIFY(peakValues1[12] == "1255329664.00");
-    // QVERIFY(peakValues1[13] == "30111416320.00");
-    // QVERIFY(peakValues1[14] == "30103683072.00");
-    // QVERIFY(peakValues1[15] == "1234160640.00");
-    // QVERIFY(peakValues1[16] == "30099625984.00");
-    // QVERIFY(peakValues1[17] == "1234094464.00");
-    QVERIFY(peakValues1[18] == "178");
-    // QVERIFY(peakValues1[19] == "116.51");
-    QVERIFY(peakValues1[20] == "0");
+    // QVERIFY(peakValues1[8] == "210.150452");
+    // QVERIFY(peakValues1[9] == "16.710");
+    // QVERIFY(peakValues1[10] == "16.603");
+    // QVERIFY(peakValues1[11] == "17.392");
+    // QVERIFY(peakValues1[12] == "0.801");
+    // QVERIFY(peakValues1[13] == "1255329664.00");
+    // QVERIFY(peakValues1[14] == "30111416320.00");
+    // QVERIFY(peakValues1[15] == "30103683072.00");
+    // QVERIFY(peakValues1[16] == "1234160640.00");
+    // QVERIFY(peakValues1[17] == "30099625984.00");
+    // QVERIFY(peakValues1[18] == "1234094464.00");
+    QVERIFY(peakValues1[19] == "178");
+    // QVERIFY(peakValues1[20] == "116.51");
+    QVERIFY(peakValues1[21] == "0");
 
     // check if labelled child values are correctly written
     vector<std::string> peakValues2;
     mzUtils::splitNew(peakString2, "," , peakValues2);
-    QVERIFY(peakValues2.size() == 21);
+    QVERIFY(peakValues2.size() == 22);
     QVERIFY(peakValues2[0] == "15");
     QVERIFY(peakValues2[1] == "210.150269@16.714417");
     QVERIFY(peakValues2[2] == "210.150269@16.714417");
     QVERIFY(peakValues2[3] == "");
-    // QVERIFY(peakValues2[4] == "testsample_3");
-    // QVERIFY(peakValues2[5] == "210.150055");
-    // QVERIFY(peakValues2[6] == "210.150101");
+    QVERIFY(peakValues2[4] == "");
+    // QVERIFY(peakValues2[5] == "testsample_3");
+    // QVERIFY(peakValues2[6] == "210.150055");
     // QVERIFY(peakValues2[7] == "210.150101");
-    // QVERIFY(peakValues2[8] == "16.719");
```

Comment:
@saifulbkhan Now that we have the method for approx equality, maybe it's time to uncomment these test cases?

Reply:
Doesn't have to be in this PR though.

Reply:
We are going to scrap these tests in favor of new ones, hopefully very soon. Or at least, they'll be modified to a large extent. But until then, I will un-comment them. It is relevant to this PR, so I am doing it here itself.

Reply:
@saifulbkhan I see this test block has been removed. Were these redundant?

Reply:
Yes, that block was testing untargeted peak-groups that were isotope-labelled. This does not make sense and test cases were the same as the above unlabelled block.